### ISSUE TITLE
Availibility Zones

### ISSUE THREAD

**Issue by adtsiko** [terraform-aws-modules/terraform-aws-rds-aurora#314]
## Is your request related to a new offering from AWS?
Is this functionality available in the [AWS provider for Terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)? See [CHANGELOG.md](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md), too.
  - Yes ✅: please list the AWS provider version which introduced this functionality

## Is your request related to a problem? Please describe.

I want to add availability zones to my rds for it is being created from a snapshot, but everytime I run `terraform apply` my rds instance recreate with no changes made towards the database terra
## Describe the solution you'd like.

A solution would be to manually set an availability zone

**Comment by antonbabenko** [comment]
Please provide a code snippet you have.

**Comment by adtsiko** [comment]
```
module "rds-aurora" {
  source  = "terraform-aws-modules/rds-aurora/aws"
  name = var.cluster_name
  version = "5.3.0"
  instance_type         = var.instance_type
  vpc_id = var.vpc_id
  database_name = var.database_name
  snapshot_identifier = var.snapshot_arn
  # insert the 12 required variables here
  vpc_security_group_ids = var.access_security_groups
  apply_immediately = false
  deletion_protection = true
  engine = "aurora-postgresql"
  engine_version = var.engine_version
  performance_insights_enabled = true
  subnets = var.subnet_ids
  tags = module.standard_tags.tags
}
```

So I'm passing in a rds snapshot from a data source

**Comment by antonbabenko** [comment]
This issue has been resolved in version 7.4.0 :tada:
